src/app.js

```diff
@@ -1,13 +1,260 @@
 // Vendor Modules
 import $ from 'jquery';
 import _ from 'underscore';
+import Trip from './app/models/trip';
+import TripList from './app/collections/tripList';
 
 // CSS
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+const TRIP_FIELDS = ['name', 'category', 'weeks', 'cost', 'about', 'continent']
+
+let tripTemplate;
+
+const loadTrips = function loadTrips(trips) {
+  const tripTable = $('#tripSection');
+  tripTable.html('');
+  trips.forEach((trip) => {
+    const generatedHTML = tripTemplate(trip.attributes);
+    tripTable.append(generatedHTML);
+  });
+}
 
 $(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+  tripTemplate = _.template($('#tripTemplate').html());
+  const singleTripTemplate = _.template($('#singleTripTemplate').html());
+  const tripList = new TripList();
+  tripList.fetch();
+
+  $('#singleTripSection').hide();
+  $('#reservationForm').hide();
+  $('#createTripSection').hide();
+  $('#numberFilter').hide();
+
+
+  $('#tripSection').on('click', 'tr', (e) => {
+    $('#singleTripSection').show();
+    $('#reservationForm').show();
+    const baseURL = 'https://ada-backtrek-api.herokuapp.com/trips/'
+    $.get(`${baseURL}${e.currentTarget.id}`, (response) => {
+      const generatedHTML = singleTripTemplate(response);
+      $('#singleTrip').html(generatedHTML);
+    }).fail(() => {
+      console.log('The post call failed');
+    });
+  });
+
+  tripList.on('sort', loadTrips, tripList);
+  tripList.on('update', loadTrips, tripList);
+
+  $('#createTripButton').on('click', () => {
+    $('#createTripSection').show();
+  });
+
+  $('#createTripForm').on('submit', (e) => {
```

Comment:
👎   Another thing that should be done with the model's `validate` method.  Also consider validations on the numeric value of the weeks and cost fields.